test/controllers/orders_controller_test.rb

```diff
@@ -1,7 +1,45 @@
 require "test_helper"
```

Comment:
Please ignore this file for now! I decided to work on this in a separate branch.